x-pack/solutions/observability/plugins/slo/common/constants.ts

```diff
@@ -65,6 +65,13 @@ export const SLI_DESTINATION_INDEX_NAME = `.slo-observability.sli-v${SLO_RESOURC
 export const SLI_DESTINATION_INDEX_PATTERN = `.slo-observability.sli-v${SLO_RESOURCES_VERSION_MAJOR}*`;
 export const SLI_INGEST_PIPELINE_INDEX_NAME_PREFIX = `.slo-observability.sli-v${SLO_RESOURCES_VERSION}.`;
 
+export const HEALTH_COMPONENT_TEMPLATE_MAPPINGS_NAME = '.slo-observability.health-mappings';
+export const HEALTH_COMPONENT_TEMPLATE_SETTINGS_NAME = '.slo-observability.health-settings';
+export const HEALTH_INDEX_TEMPLATE_NAME = '.slo-observability.health';
+export const HEALTH_INDEX_TEMPLATE_PATTERN = `.slo-observability.health-*`;
+export const HEALTH_INDEX_NAME = `.slo-observability.health-v${SLO_RESOURCES_VERSION}`;
+export const HEALTH_INDEX_PATTERN = `.slo-observability.health-v${SLO_RESOURCES_VERSION_MAJOR}*`;
```

Comment:
maybe avoid . naming to not fall into fixing issues with dot prefix naming, othewise you will have to get this added into those regex patterns here https://github.com/elastic/elasticsearch/blob/main/modules/dot-prefix-validation/src/main/java/org/elasticsearch/validation/DotPrefixValidator.java#L74

Reply:
actually maybe not needed, since it's created with internal kibana system user.

Reply:
At first i used a non-dot prefixed index but got yelled at by es when trying to create it while using the kibana system user. I don't know why, do you?

Reply:
Oh yes thanks I forgot about that whitelisting in es...